custom_components/opnsense/sensor.py

```
@@ -41,322 +42,370 @@
     )
-    cem.process_entities()
+
+    entities: list = []
+    for result in results:
+        if isinstance(result, list):
+            entities += result
+        else:
+            _LOGGER.error(
+                f"Error in sensor async_setup_entry. {result.__class__.__qualname__}: {result}"
+            )
+    _LOGGER.debug(f"[sensor async_setup_entry] entities: {len(entities)}")
```

Ruff

Logging statement uses f-string

(G004)